packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-strategy.ts

```diff
@@ -40,7 +40,7 @@ export default class AmazonPayV2PaymentStrategy implements PaymentStrategy {
 
         await this._amazonPayV2PaymentProcessor.initialize(paymentMethod);
 
-        if (this._isReadyToPay(features, paymentToken) && amazonpay?.editButtonId) {
+        if (this._isReadyToPay(paymentToken) && amazonpay?.editButtonId) {
```

Comment:
Can we please add unit test to cover this behaviour? Seems like something we didn't do last time as well.